include/avt_vimba_camera/mono_camera.h

```diff
@@ -43,19 +43,22 @@
 #include <camera_info_manager/camera_info_manager.h>
 #include <image_transport/image_transport.h>
 #include <dynamic_reconfigure/server.h>
-
+#include <cav_msgs/SystemAlert.h>
+#include <cav_msgs/DriverStatus.h>
 #include <string>
 
 namespace avt_vimba_camera {
 class MonoCamera {
  public:
   MonoCamera(ros::NodeHandle& nh, ros::NodeHandle& nhp);
   ~MonoCamera(void);
-
+ void time_compare();
```

Comment:
This function does not only compare the time. I would use a better name like updateCameraStatus.
Also in general, function and class method names are camelCased, and arguments are under_scored.